themes/default/content/resources/getting-started-infrascructure-as-code-java/index.md

```diff
@@ -0,0 +1,94 @@
+---
+# Name of the webinar.
+title: "Get Started with Infrastructure as Code on Azure with Java"
+meta_desc: "Learn the basics of Azure and Pulumi with a hands-on lab that will take you from deploying a simple static website to deploying a Spring Boot application."
+
+# A featured webinar will display first in the list.
+featured: false
+
+# If the video is pre-recorded or live.
+pre_recorded: false
+
+# If the video is part of the PulumiTV series. Setting this value to true will list the video in the "PulumiTV" section.
+pulumi_tv: false
+
+# The preview image will be shown on the list page.
+preview_image: ""
+
+# Webinars with unlisted as true will not be shown on the webinar list
+unlisted: false
+
+# Gated webinars will have a registration form and the user will need
+# to fill out the form before viewing.
+gated: true
+
+# The layout of the landing page.
+type: webinars
+
+# External webinars will link to an external page instead of a webinar
+# landing/registration page. If the webinar is external you will need
+# set the 'block_external_search_index' flag to true so Google does not index
+# the webinar page created.
+external: false
+block_external_search_index: false
+
+# data for Google Events
+event_data:
+  name: "Get Started with Infrastructure as Code on Azure with Java"
+  start_date: 2022-11-22T09:00:00-08:00
```

Comment:
@zchase - is there a special way that we are handling daylight savings time? I just changed this from -7:00 to -8:00 to get the times to line up right. is there a better way I should be doing this?